aap_compose_dev.yaml

```diff
@@ -200,12 +228,91 @@ services:
+      - worker
+    volumes:
+      - "etc_pulp_certs:/etc/pulp/certs"
+      - "var_lib_pulp:/var/lib/pulp"
+      - ".:/src/galaxy_ng"
+    environment:
+      <<: *common-env
+    user: root
+    command: |
+      bash -c "
+        while [[ ! -f /var/lib/pulp/.migrated ]]; do
+          echo 'Waiting for migrations ...';
+          sleep 2;
+        done;
+        while [[ ! -f /etc/pulp/certs/ansible-sign.key ]]; do
+          echo 'Waiting for signing key';
+          sleep 2;
+        done;
+
+        echo '#> STEP: Scheduling Resource Sync Task.';
+        pulpcore-manager task-scheduler --id dab_sync --interval 15 --path "galaxy_ng.app.tasks.resource_sync.run";
+        curl -s -u admin:admin http://api:24817/api/galaxy/pulp/api/v3/task-schedules/?name=dab_sync | python -m json.tool;
+        echo 'DONE! <#';
+
+        echo '#> STEP: Import GPG Keys for signing service creation.';
+        gpgconf --kill gpg-agent && gpg --batch --no-default-keyring --import /etc/pulp/certs/ansible-sign.key;
+        (echo 5; echo y; echo save) | gpg --command-fd 0 --no-tty --no-greeting -q --edit-key 'FB8B3F2D24BCAF7EFDF793A9F37575C52D4F16F3' trust;
+        gpg --list-secret-keys;
+        echo 'DONE! <#';
+        echo '#> STEP: Creating signing services';
+        pulpcore-manager add-signing-service ansible-default /var/lib/pulp/scripts/collection_sign.sh F37575C52D4F16F3;
+        pulpcore-manager add-signing-service container-default /var/lib/pulp/scripts/container_sign.sh F37575C52D4F16F3 --class container:ManifestSigningService;
+        # add-signing-service is not idempotent, so the note below.
+        echo 'NOTE!!! CommandError: duplicate key value, above is NOT A PROBLEM if 2 signing services are returned from API below:';
+        curl -s -u admin:admin http://api:24817/api/galaxy/pulp/api/v3/signing-services/?fields=name,script,pubkey_fingerprint | python -m json.tool;
+        echo 'DONE! <#';
+        echo '#> STEP: Setting repository public key for signature upload verification'
+        pulpcore-manager set-repo-keyring --repository staging --publickeypath /etc/pulp/certs/ansible-sign-pub.gpg -y;
+        pulpcore-manager set-repo-keyring --repository published --publickeypath /etc/pulp/certs/ansible-sign-pub.gpg -y;
+        echo 'DONE! <#';
+
+        echo '#> STEP: Installing dev tools';
+        /venv/bin/pip3.11 install ipython ipdb django-extensions;
+        echo 'DONE! <#';
+        echo ' ';
+
+        echo '###################### API ROOT ##############################';
+        curl -s http://api:24817/api/galaxy/ | python -m json.tool;
+
+        echo '######################## READY ###############################';
+        echo ' ';
+        echo 'API: http://localhost:5001/api/galaxy/v3/swagger-ui/';
+        echo 'Django Admin CLI: docker compose -f aap_compose_dev.yaml exec manager pulpcore-manager';
+        echo 'Settings list: docker compose -f aap_compose_dev.yaml exec manager dynaconf list';
+
```

Comment:
After systems are up this is the output,
I thought about having `Make` targets for those but I think just using socker compose exec directly might better practice if we list useful cmds.

```bash
manager-1     | DONE! <#
manager-1     |  
manager-1     | ###################### API ROOT ##############################
api-1         | 2024-10-07 15:31:16,388 INFO ansible_base.jwt_consumer.common.auth: X-DAB-JW-TOKEN header not set for JWT authentication
manager-1     | {
manager-1     |     "available_versions": {
manager-1     |         "v3": "v3/",
manager-1     |         "pulp-v3": "pulp/api/v3/"
manager-1     |     },
manager-1     |     "server_version": "4.11.0dev",
manager-1     |     "galaxy_ng_version": "4.11.0dev",
manager-1     |     "galaxy_ng_commit": "",
manager-1     |     "galaxy_importer_version": "0.4.24",
manager-1     |     "pulp_core_version": "3.49.15",
manager-1     |     "pulp_ansible_version": "0.21.7",
manager-1     |     "pulp_container_version": "2.19.3",
manager-1     |     "ansible_base_version": "2024.9.27.0.dev13+g5f1ca89"
manager-1     | }
manager-1     | ######################## READY ###############################
manager-1     |  
manager-1     | API: http://localhost:5001/api/galaxy/v3/swagger-ui/
manager-1     | Django Admin CLI: docker compose -f aap_compose_dev.yaml exec manager pulpcore-manager
manager-1     | Settings list: docker compose -f aap_compose_dev.yaml exec manager dynaconf list

```